packages/server/test/e2e/6_non_proxied_spec.ts

```diff
@@ -0,0 +1,15 @@
+const e2e = require('../support/helpers/e2e')
+const Fixtures = require('../support/helpers/fixtures')
+
+describe('e2e non-proxied spec', () => {
+  e2e.setup()
+
+  e2e.it('passes', {
```

Comment:
this e2e test only needs to run "once" in a single browser since it hard codes launching chrome

Reply:
Maybe just run it in `electron`?

Reply:
sure, save a few cents on the CI bill :smile: